jedi/inference/compiled/subprocess/functions.py

```diff
@@ -69,18 +102,6 @@ def _test_print(inference_state, stderr=None, stdout=None):
         sys.stdout.flush()
 
 
-def _get_init_path(directory_path):
```

Comment:
This is an unrelated change (i.e. not related to #1759) since I was just removing some dead code.